content/tutorials/drift-detection-and-remediation/index.md

```diff
@@ -0,0 +1,224 @@
+
+1. Go to [GitHub](https://github.com/new) and create a new repository manually.
+2. Copy the repository URL provided by GitHub.
+3. Replace the origin with your new GitHub repo using the following commands:
+
+```bash
+git init
+git add .
+git commit -m "Initial commit for my Pulumi Drift tutorial"
+git branch -M main
+git remote add origin https://github.com/your-username/learn-pulumi-drift-detection.git
+git push -u origin main
+```
+
+## Configure Drift Detection
+
+Now you will return to your browser to set your deployment configuration source control settings. Navigate to your stack `Settings` and `Deploy` tab. Next, configure the required Deployment settings, including your source control.
+
+1. Add your GitHub organization/repository and the branch you created.
+
+2. Add your Secret environment variables, including `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
+
+{{% notes type="info" %}}
+Secret environment variables, such as `AWS_SECRET_ACCESS_KEY`, are encrypted end-to-end with Pulumi and can be set on each stack. However, by creating an [environment](/docs/pulumi-cloud/esc) with Pulumi ESC, you can centralize secrets and set up OIDC for secure authentication. This allows you to manage and share sensitive configuration data across multiple stacks efficiently.
+{{% /notes %}}
+
+Click **Save deployment configuration**.
+
+<video autoplay loop muted playsinline>
+    <source src="pulumi-configure-drift.mp4" type="video/mp4">
+    Your browser does not support the video tag.
+</video>
+
+Once you have configured Pulumi Deployments, open the Actions menu and choose `Detect Drift`, then click `Deploy` to validate your stack, program and deployments are all running before we introduce drift.
+
+![Screenshot of the Pulumi Cloud console to run a Drift Detection](./pulumi-detect-drift-action.png)
+
+After the drift detection is complete, you will see a new Drift run on the `Drift` tab timeline. Since this is the initial check and no deviations have been introduced, you will not receive any notifications indicating that drift has been detected, nor will you see any resources with changes.
+
+## Introduce drift
+
+To see Drift Detection in action, you will now introduce drift by manually making changes to your newly created EC2 resource, in this case by adding a new tag.
+
+### Add a tag to EC2 using AWS CLI
+
+To add a tag via the AWS CLI you will first need to retrieve the EC2 Instance ID before you can add a tag. Use the the following command:
+
+```bash
+   aws ec2 describe-instances --filters "Name=tag:Name,Values=webserver" --query "Reservations[*].Instances[*].InstanceId" --output text
+```
+
+Now, add the tag with the key `Description` and the value `Pulumi Drift Detection`:
+
+```bash
+   aws ec2 create-tags --resources <instance-id> --tags Key=Description,Value="Pulumi Drift Detection"
+```
+
+Replace `<instance-id>` with the actual instance ID of your EC2 instance.
+
+### Add a tag to EC2 using the AWS console
```

Comment:
cant you shove this stuff into a tabbed chooser module? otherwise you need to make it clear these are the same instructions executed 2 different ways..

Reply:
Agree -- we don't need both of these. Suggest just using the console version. It's simpler.

Reply:
I think I requested in the last edit to add a CLI version since its actually faster to do that since you already have the credentials in the CLI

Reply:
dropped the console version